examples/cmri_mpca/tutorial.ipynb

```diff
@@ -2,15 +2,23 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
```

Comment:
Find a way to remove this consistently to reduce review burden and show changes more clearly.